prebid-server/features/pbs-fpd.md

```diff
@@ -27,12 +27,12 @@ First Party Data in the OpenRTB JSON in several places:
 {: .table .table-bordered .table-striped }
```

Comment:
The table goes way outside the page boundaries. (Not caused by this PR, it's like this now.) Didn't atre.net do something to fix this on other pages? I looked around a bit and didn't find anything.

Reply:
IIRC, the only thing we can do about the wide tables is break up whatever long "word" in each column is causing the table to not be able to wrap that column. 

- ext.prebid.data.bidders[]
- config.ortb2.user.ext.data.ATTR
- addBidderToAccessControlList()

I tweaked it to change the period after "config" to a space. Every little bit helps?